### ISSUE TITLE
react native sunmi card reader

### ISSUE THREAD

**Issue by mohmmadmoussa1988** [januslo/react-native-sunmi-inner-scanner#12]
Dear Januslo,

Could you please tell me where can i find react native sunmi card reader ?

**Comment by januslo** [comment]
Card reader? Sorry I don’t known about that

发自我的 iPhone

**Comment by mohmmadmoussa1988** [comment]
yes Dear i noticed that you prepared react-native module for opening the
scanner and the printer, can we cooperate on preparing a module for
activate Card reader, IC Activity ? i need your help and ready to pay for
it.

**Comment by januslo** [comment]
Plz send more information to januslo6211@icloud.com / shadow557@163.com for further discussion 

发自我的 iPhone

**Comment by mohmmadmoussa1988** [comment]
Hello,

I sent an email but no reply untill now.

this is my sent email :



mohammad moussa <webdeveloper.mohammad@gmail.com>
Sat, Aug 24, 1:18 PM (3 days ago)
to januslo6211, shadow557
Dear Janulso6211,

Im a react-native developer looking for a module to pay using Sunmi p2 List
:

https://www.sunmi.com/en/P2lite/

Please check the below video :

https://www.youtube.com/watch?v=-cYvgKcnzIU

I have searched for module to be installed on the sunmi device and found
modules made by you on GitHub, if you check the you tube video you will
notice the pay app done with sunmi, i need something similar built with
react-native to be installed on sunmi device, are you able to do something
similar for me ?

Thank you

**Comment by januslo** [comment]
I got this, plz wait.

发自我的 iPhone

**Comment by gaikwadamolraj** [comment]
Hey, 

i have sumi D3 mini screen and it has a nfc reader. I have my android app built and doing the order. Now i want to initiate payment and read the card reader from the customer and call the stripe payment gateway apis. I have code for the stripe payment gateway but i am not able to get the nfc card reader details. 

Please help me how to get the details share the documentation for the same.

**Comment by LuisCarcamo21** [comment]
Hello, as of today, is there any library to implement the card reader with React Native on Sumni P2-P3?
